game_engine.properties

```diff
@@ -4,15 +4,3 @@
 ## first.
 
 engine_version = 1.9.0.0
-
-## URL where we will download the list of available maps
-map_list_file = https://raw.githubusercontent.com/triplea-game/triplea/master/triplea_maps.yaml?raw=true
-## Example that would use a local file config file, useful for map building or development
-# map_list_file = triplea_maps.yaml
-
-
-## Location where we download latest lobby description file, this tells us where the latest lobby is located
-lobby_properties_file_url = https://raw.githubusercontent.com/triplea-game/triplea/master/lobby_server.yaml?raw=true
```

Comment:
once we have the back removed, we are just left with two URLs. IT actually turns out we may as well hardcode them. They are now system settings, and we have a default value for them, if a user wants to change them, they can do so via the UI. No need for a  props file. If we want to release to update the values in this file, it is just as well the values be in code.